Rogue-Ape-Studios/Crusader
# 🔧 Fix/repair prompt

## Content

Adjusted the .gitattributes file to prevent git from clobbering the line endings to prevent the FMOD repair prompt from appearing. 

## Changes

### Updated
`.gitattributes` : Was updated. Added 2 entries to prevent git from changing the line endings when pulling.


## Testing

The feature / Bugfix can be testing by following these steps:

1. Click repair once
2. See result
